build.js

```diff
@@ -61,7 +62,7 @@ if (shell.exec(`rollup -c rollup.es.config.js -i ${NPM_DIR}/${PACKAGE}.js -o ${F
 }
 
 shell.echo(`Produce ESM5/FESM5 versions`);
-shell.exec(`ngc -p ${OUT_DIR}/tsconfig-build.json --target es5 -d false --outDir ${OUT_DIR_ESM5} --sourceMap`);
+shell.exec(`ngc -p ${OUT_DIR}/tsconfig-build.json --target es5 -d false --outDir ${OUT_DIR_ESM5_ABS} --sourceMap`);
```

Comment:
Otherwise there will be following:
> Produce ESM5/FESM5 versions
> Error: expected "dist/package/esm5/src/services/sum.service.js.map" to be absolute